pages/instructorAssessmentManualGrading/instructorAssessmentManualGrading.sql

```diff
@@ -1,20 +1,73 @@
--- BLOCK select_instance_questions_manual_grading
-SELECT DISTINCT ON (iq.id)
-    iq.*,
-    s.graded_at,
+-- BLOCK select_questions_manual_grading
+WITH issue_count AS (
+    SELECT
+        q.id AS question_id,
+        count(*) AS open_issue_count
+    FROM
+        issues AS i
+        JOIN questions AS q ON (q.id = i.question_id)
+    WHERE
+        i.assessment_id = $assessment_id
+        AND i.course_caused
+        AND i.open
+    GROUP BY q.id
+),
+question_scores AS (
+    SELECT
+        aq.question_id,
+        avg(aq.mean_question_score) AS question_score
+    FROM
+        assessment_questions AS aq
+    WHERE
+        aq.assessment_id = $assessment_id
+    GROUP BY
+        aq.question_id
+)
+SELECT
+    aq.*,
+    q.qid,
+    q.title,
     q.id AS question_id,
-    q.title AS question_title,
-    aq.max_points,
-    ai.id AS assessment_instance_id,
-    qo.question_number
+    admin_assessment_question_number(aq.id) as number,
+    ag.number AS alternative_group_number,
+    ag.number_choose AS alternative_group_number_choose,
+    (count(*) OVER (PARTITION BY ag.number)) AS alternative_group_size,
+    z.title AS zone_title,
+    z.number AS zone_number,
+    z.number_choose as zone_number_choose,
+    (lag(z.id) OVER (PARTITION BY z.id ORDER BY aq.number) IS NULL) AS start_new_zone,
+    (lag(ag.id) OVER (PARTITION BY ag.id ORDER BY aq.number) IS NULL) AS start_new_alternative_group,
+    coalesce(ic.open_issue_count, 0) AS open_issue_count,
+    question_scores.question_score AS avg_question_score_perc,
+    z.max_points AS zone_max_points,
+    (z.max_points IS NOT NULL) AS zone_has_max_points,
+    z.best_questions AS zone_best_questions,
+    (z.best_questions IS NOT NULL) AS zone_has_best_questions,
```

Comment:
The code above looks like it was copied from `instructorAssessmentQuestions`? It has a lot of stuff in there to render zones, max points, best questions, etc. Do we want all this for the manual grading overview page? I tend to think that we should just show a simple list of questions with information about what still needs grading, with no extra extraneous info.

Reply:
Removed zones and assessment groups, though I kept the assessment group numbering.